Add text description to non-decorative pictures

podliashanyk opened the pull request:
Information bearing pictures must have a text description.

Following requirements are satisfied:

- WCAG 2.0 - 1.1.1
- WCAG 2.0 - 1.4.5

podliashanyk committed:
Add text description to non-decorative pictures

Information bearing pictures must have a text description.

lunkwill42 commented on layouts/index.html:
```diff
@@ -135,28 +135,49 @@ <h1 class="super-heading">About NAV</h1>
                                         <div class="col-md-12">
-                                            <img class="" src="/image/nav1.png" alt="...">
+                                            <img class="" src="/image/nav1.png"
+                                                 alt="Screenshot of the NAV's frontpage."
```
```suggestion
                                                 alt="Screenshot of NAV's front page."
```

lunkwill42 commented on layouts/index.html:
```diff
@@ -135,28 +135,49 @@ <h1 class="super-heading">About NAV</h1>
-                                            <img class="" src="/image/nav1.png" alt="...">
+                                            <img class="" src="/image/nav1.png"
+                                                 alt="Screenshot of the NAV's frontpage."
+                                                 aria-describedby="Frontpage of NAV.
+                                                 On the left side there are the 'Getting started with NAV'-section and a top portion of some graph.
+                                                 On the right side there are following sections: NAV blog, Status, Action map"
+                                            >
```
```suggestion
                                                 aria-describedby="Front page of NAV.
                                                 On the left side there is the 'Getting started with NAV'-section and a top portion of a graph.
                                                 On the right side there are the following sections: NAV blog, Status, Action map"
                                            >
```

lunkwill42 commented on layouts/index.html:
```diff
@@ -135,28 +135,49 @@ <h1 class="super-heading">About NAV</h1>
                                         <div class="col-md-12">
-                                            <img class="" src="/image/nav4.png" alt="...">
+                                            <img class="" src="/image/nav4.png"
+                                                 alt="Screenshot of the NAV's Geomap page."
```
```suggestion
                                                 alt="Screenshot of NAV's Geomap page."
```

lunkwill42 commented on layouts/index.html:
```diff
@@ -135,28 +135,49 @@ <h1 class="super-heading">About NAV</h1>
+                                                 aria-describedby="NAV's Geomap page.
+                                                 Most of the picture is a map of a street. There is line from one building to another.
+                                                 The line is half grey and half green.
+                                                 Below the map following metrics is displayed: CPU load, netboxes, Link load, Link capacity."
```
```suggestion
                                                 Below the map the following metrics are displayed: CPU load, netboxes, Link load, Link capacity."
```

lunkwill42 commented on layouts/index.html:
```diff
@@ -135,28 +135,49 @@ <h1 class="super-heading">About NAV</h1>
+                                                 alt="Screenshot showing temperature graphs of the sensors of some device."
+                                                 aria-describedby="Part of the page showing temperature graphs of the sensors of some device.
+                                                 The IP address of the device is blurred out.
+                                                 There are 3 graphs, each shows a current temperature of the sensor and a graph over temperature change below.
```
```suggestion
                                                 There are 3 graphs, each shows a current temperature of the sensor and a graph of temperature change below.
```

lunkwill42 commented on layouts/index.html:
```diff
@@ -135,28 +135,49 @@ <h1 class="super-heading">About NAV</h1>
                                         <div class="col-md-12">
-                                            <img class="" src="/image/nav5.png" alt="...">
+                                            <img class="" src="/image/nav5.png"
+                                                 alt="Screenshot of graph of the traffic over some port."
```
```suggestion
                                                 alt="Screenshot of graph of the traffic on some port."
```

lunkwill42 commented on layouts/index.html:
```diff
@@ -135,28 +135,49 @@ <h1 class="super-heading">About NAV</h1>
+                                            <img class="" src="/image/nav5.png"
+                                                 alt="Screenshot of graph of the traffic over some port."
+                                                 aria-describedby="The graph is called 'Port details'.
+                                                 It shows in- and out-traffic over some port measured in bites per second."
```
```suggestion
                                                 It shows in- and out-going traffic on some port, measured in bits per second."
```

lunkwill42 reviewed (changes requested):
Nice :)  I only have proofreading suggestions.

podliashanyk committed:
Update layouts/index.html

Co-authored-by: Morten Brekkevold <morten.brekkevold@sikt.no>
Update layouts/index.html

Co-authored-by: Morten Brekkevold <morten.brekkevold@sikt.no>
Update layouts/index.html

Co-authored-by: Morten Brekkevold <morten.brekkevold@sikt.no>
Update layouts/index.html

Co-authored-by: Morten Brekkevold <morten.brekkevold@sikt.no>
Update layouts/index.html

Co-authored-by: Morten Brekkevold <morten.brekkevold@sikt.no>
Update layouts/index.html

Co-authored-by: Morten Brekkevold <morten.brekkevold@sikt.no>
Update layouts/index.html

Co-authored-by: Morten Brekkevold <morten.brekkevold@sikt.no>

podliashanyk committed:
Fix broken aria reference error

lunkwill42 reviewed (approved):
:+1:
